priv/repo/migrations/20210812234934_hotspot_nullable_fields.exs

```diff
@@ -0,0 +1,39 @@
+defmodule Console.Repo.Migrations.HotspotNullableFields do
+  use Ecto.Migration
+
+  def up do
+    Ecto.Adapters.SQL.query!(
+      Console.Repo, """
+        DELETE FROM hotspots WHERE
+          height IS NULL OR
+          location IS NULL OR
+          lat IS NULL OR
+          lng IS NULL OR
+          short_state IS NULL OR
+          short_country IS NULL OR
+          long_city IS NULL
+      """)
+    
+    alter table("hotspots") do
+      modify :height, :integer, null: true
+      modify :location, :string, null: true
+      modify :lat, :decimal, null: true
+      modify :lng, :decimal, null: true
+      modify :short_state, :string, null: true
+      modify :short_country, :string, null: true
+      modify :long_city, :string, null: true
+    end
```

Comment:
we need to add a uniqueness check for hotspot_address, my fault for forgetting in my previous PR

Reply:
"hotspots_address_index" needs to be an UNIQUE index in the db